modules/weko-records-ui/weko_records_ui/pdf.py

```diff
@@ -429,45 +445,57 @@ def get_oa_policy(activity_id):
             metadata_dict["affiliation"])
     ]
     metadata = '\n'.join(metadata_list)
-    metadata_lfnum = int(metadata.count('\n'))
-
-    for item in metadata_list:
-        metadata_lfnum += int(get_east_asian_width_count(item)
-                              ) // max_letters_num
 
     # Get url
     url = get_url(pid.pid_value)
-    url_lfnum = int(get_east_asian_width_count(url)) // max_letters_num
 
-    # Save top coordinate
-    top = pdf.y
     # Calculate x position of next cell
     offset = pdf.x + w1
+
+    # Save top coordinate
+    top = pdf.y
+    # Reset y coordinate
+    pdf.y = top
+    # Move to computed offset
+    pdf.x = offset
+    pdf.multi_cell(w2, meta_h, metadata, 1, 'L', False)
+    # Get height w2 and calculate line number of metadata
+    metadata_lfnum = 0
+    height = pdf.y - top
+    if height > 0:
+        metadata_lfnum = int(height / meta_h)
+    # Reset y coordinate
+    pdf.y = top
     pdf.multi_cell(w1,
                    meta_h,
-                   lang_data["Title"]["METADATA"]
-                   + '\n' * (metadata_lfnum + 1),
+                   lang_data["Title"]["METADATA"] + '\n' * metadata_lfnum,
                    1,
                    'C',
                    True)
+
+    # Save top coordinate
+    top = pdf.y
```

Comment:
This op should depence on this process:
`pdf.multi_cell(`

Reply:
It is used to create another content